io.github.mrquantumoff.mcmodpackmanager.metainfo.xml

```diff
@@ -31,6 +31,11 @@ NOTE: This app and its developer are not affiliated with Mojang Studios.
   <keyword>Minecraft Modpack Manager</keyword>
 </keywords>
 <releases>
+
+<release date="2022-21-05" version="2.5.1"></release>
+
+<release date="2022-21-05" version="2.5.0"></release>
+
```

Comment:
these dates are invalid. There is no month 21.

Reply:
Yeah, I'm not used to this format so I mixed those up